api/proto/v1/common.proto

```diff
@@ -23,3 +23,37 @@ message ReturnMessage {
   ReturnCode return_code = 1;
   string return_message = 2;
 }
+
+message InternalServiceInfo {
+  OperationType operation_type = 1;
+  string id = 2;
+  string name = 3;
+  string cmd = 4;
+  string url = 5;
+  repeated string parameters = 6;
+  repeated uint32 return_code = 7;
+  repeated string return_string = 8;
+  string when_to_run = 9;
+  string where_to_run = 10;
+}
+
+message InternalComputeInfo {
+  OperationType operation_type = 1;
+  string id = 2;
+  string name = 3;
+  string ip = 4;
+  string mac = 5;
+  string veth = 6;
+}
+
+message InternalSubnetInfo {
+  string subnet_id = 1;
+  string subnet_cidr = 2;
+  string subnet_gw = 3;
+  uint32 number_vms = 4;
+}
+
+message InternalVpcInfo {
+  string vpc_id = 1;
```

Comment:
We need `vpc_cider` and maybe `vpc_gw` if we want to specific our own

Reply:
@yanmo96 does alcor require vpc_cidr and vpc_gw? In what scenario we need these two parameters?